tutorials/plot_object_annotations.py

```diff
@@ -75,19 +78,25 @@
 mne.viz.plot_events(events, raw.info['sfreq'], raw.first_samp, color=color,
                     event_id=event_id)
 
-# Create some annotations specifying onset, duration and description
+###############################################################################
+# Next, we're going to find instances of eye blinks and turn them into
+# :class:`Annotations <mne.Annotations>`:
```

Comment:
FYI you can just do:
```
:class:`~mne.Annotations`
```
and it will look the same (it keeps only what follows the last `.`)